package.json

```diff
@@ -35,6 +35,7 @@
   "devDependencies": {
     "@types/node": "^16.0.0",
     "json-schema-to-typescript": "^10.0.3",
+    "minecraft-packets": "github:PrismarineJS/node-minecraft-packets",
```

Comment:
Why ?

Reply:
This is used in a test

Reply:
I mean why not depend on the version published on npm ?

Reply:
I published this, v1.4.0 has what you pr'd